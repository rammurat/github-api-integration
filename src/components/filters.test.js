import React from 'react'
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux'

import Filters from './filters'

const store = {
    subscribe: () => { },
    dispatch: () => { },
    getState: () => ({
        posts: {
            initialConfig: {}
        }
    })
}

// @ts-ignore
it('renders correctly', () => {
    const tree = renderer
        // @ts-ignore
        .create(<Provider store={store}><Filters /></Provider>)
        .toJSON();
    // @ts-ignore
    expect(tree).toMatchSnapshot();
});