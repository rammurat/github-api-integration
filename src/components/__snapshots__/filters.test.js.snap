// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <form>
    <h1>
      Filters:
    </h1>
    <div
      className="form-group"
    >
      <label
        htmlFor="language-type"
      >
        Language Type:
      </label>
      <select
        className="form-control"
        id="language-type"
        name="language-type"
        onChange={[Function]}
      />
    </div>
    <div
      className="form-group"
    >
      <label
        htmlFor="sort-by"
      >
        Sort by stars:
      </label>
      <select
        className="form-control"
        id="sort-by"
        name="sort-by"
        onChange={[Function]}
      />
    </div>
  </form>
</div>
`;
